.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off",
+      "indent": [2, "tab"],
+      "no-tabs": 0,
+      "import/no-cycle": "off",
+      "no-restricted-globals": "off",
+      "no-alert": "off"
```

Comment:
- [ ] Please use the same lint config rules in this [link](https://github.com/microverseinc/linters-config/tree/master/javascript). It's important you use the same eslint rules used by fellow students too.